src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -514,7 +516,9 @@ public void fetch(String remoteName, RefSpec... refspec) throws GitException, In
         String url = getRemoteUrl(remoteName);
         if (url == null)
             throw new GitException("remote." + remoteName + ".url not defined");
+
```

Comment:
Please remove this whitespace change from the pull request.  There are many pull requests which need to be evaluated for the git client plugin.  White space changes tend to cause unnecessary merge conflicts with other pull requests.